BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -508,19 +539,40 @@ internal extension GiniBankNetworkingScreenApiCoordinator {
     func determineIfPaymentHintsEnabled(for extractionResult: ExtractionResult) -> Bool {
         let globalPaymentHintsEnabled = giniBankConfiguration.paymentHintsEnabled
         let clientPaymentHintsEnabled = GiniBankUserDefaultsStorage.clientConfiguration?.paymentHintsEnabled ?? false
-        let documentIsPaid = isDocumentMarkedAsPaid(extractionResult)
```

Comment:
[Clarification]: For making sure, we don't use anymore `isDocumentMarkedAsPaid(extractionResult)` is it correct?